openapi/core-data.yaml

```diff
@@ -562,17 +562,17 @@ components:
             resourceName: "Int8"
             profileName: "Random-Integer-Device"
             valueType: "Int8"
-            value: "75"
+            value: 75
           - deviceName: "Random-Float-Device"
             resourceName: "Float32"
             profileName: "Random-Float-Device"
             valueType: "Float32"
-            value: "112.2"
+            value: 112.2
           - deviceName: "Random-String-Device"
             resourceName: "String"
             profileName: "Random-String-Device"
             valueType: "String"
-            value: null
+            value: ""
```

Comment:
The aggregate value is changed to empty instead of null for non-numeric value types after using `NumericReading` model.